### ISSUE TITLE
Docs not entirely clear about config requirements

### ISSUE THREAD

**Issue by okdistribute** [township/township#4]
I'm not sure what you mean by 'secret' -- does it have to be a particular kind of key? The example in `township-example` is using some EC keys, not entirely sure how those are generated or used and that seems undocumented. 

Is postmark integration optional? 

Thanks for all your hard work, this is a great interface!

**Comment by sethvincent** [comment]
Agreed. These repos need some serious docs. I'm planning to do a minidocs + ecosystem-docs site for township stuff, just need to set aside some time!

I'm also not sure how those keys were generated, if you mean these: https://github.com/township/township/blob/master/test/test.js#L19

Yeah, I'd like postmark to be optional. Also super open to alternatives. I went with that service initially just because it's easy to set up really fast.
